cmd/convox/main.go

```diff
@@ -112,12 +112,11 @@ func currentRack(c *cli.Context) string {
 func rackClient(c *cli.Context) *client.Client {
 	host, password, err := currentLogin()
 	if err != nil {
-		stdcli.Error(err)
-		return nil
```

Comment:
i get why this was changed like this, that function is called everywhere.. Should it just return an error "the go way?"